static/editjs.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function(){
+    $("button#tweetbutton").click(function(){
+        tweet_entry();
+    });
+    $("button#editbutton").click(function(){
+      add_entry_to_db();
+    });
+});
+
+function add_entry_to_db() {
+    $.post("", {'title' : $("[name=title]").val(), 'text' : $("[name=text]").val()});
+}
```

Comment:
Interesting.  You are relying on the fact that the ID is already in the URL to ensure that you end up editing the same post.  That works.